src/app/(app)/page.tsx

```diff
@@ -13,9 +14,9 @@ export default async function Home() {
     })
 
     return (
-        <>
+        <><NavBar/>
```

Comment:
It's advisable to have this in layout.tsx. This will let us use the same layout for the sub-routes. This is true for the Footer too (that I forgot to mention back then sadly)